package.json

```diff
@@ -23,7 +20,7 @@
     "jscs": "./node_modules/jscs/bin/jscs *.js api cache clock config db jobQueue migrations realtime tests utils view worker",
     "lint": "eslint view/admin",
     "migratedb": "node db/migrate.js",
-    "postinstall": "NODE_ENV=production gulp browserifyViews && gulp movecss && gulp movesocket",
+    "postinstall": "NODE_ENV=production gulp browserifyViews && gulp movecss && . ./scripts/movesocket.sh",
```

Comment:
When calling the script, the . before PATH calls the script in the current shell http://superuser.com/questions/46139/what-does-source-do